### PR TITLE
 Add Listing functionalities for Resumes

### LINKED ISSUE #4: List all resumes related to a single user.

### LINKED ISSUE #14: add resume url to the output
forgot to ad dit

### PR DESCRIPTION
**Pull request by eliashomsi**
List all resumes related to a single user.
Get a specific resume for a user given the candidate key user_id, revision, title
Add comprehensive tests for users.

Fixes #4 
Fixes #14

### CODE REVIEW

File: .ruby-version
Comment (eliashomsi) [inline .ruby-version:1]: I was suggested to change this by the compiler

File: app/controllers/application_controller.rb
Comment (eliashomsi) [inline app/controllers/application_controller.rb:1]: changing the type of the controller since we are simply an API and not a web applciation

File: app/models/resume.rb
Comment (eliashomsi) [inline app/models/resume.rb:13]: this catched lots of errors 4 me

File: test/fixtures/resumes.yml
Comment (eliashomsi) [inline test/fixtures/resumes.yml:1]: these are fixtures used for tests.

File: test/models/resume_test.rb
Comment (eliashomsi) [inline test/models/resume_test.rb:1]: this part still needs work. however, resume creation was tested comprehensivly using integration tests.

**Review by eliashomsi — COMMENTED** [review]
Waiting for approval.

**Review by abbasyadollahi — APPROVED** [review]

**Pull request merged**
